rewards/README.md

```diff
@@ -4,40 +4,80 @@ cover: ../.gitbook/assets/cover/rewards_banner.png
+
+Active Rewards focuses on incentivizing the curation and publishing of high-quality data assets and DataNFTs within the Ocean Ecosystem. 
+
+We have used battle-tested contracts that safeguard billions (veCRV). With this model there is no liquidity risk. You can’t lose your OCEAN through Impermanent-Loss (IL). It is merely locked.
+
+## Introduction
+
+[Farming 101](df-intro.md) introduces Data Farming, the various reward systems, how they work, and how to access them. By the end of the document, you should feel more familiar with Data Farming concepts and having a better idea of what subject you might want to learn about next.
 
 ## veOCEAN
 
-Learning about [veOCEAN](veocean.md) will help you answer the question "What is the purpose of holding veOCEAN?" & give insights on how veOCEAN (vote-escrowed OCEAN) works. It will teach you everything you need to know about why it exists and how it works.
+Learning about [veOCEAN](df-veocean.md) will help you answer the question "What is the purpose of veOCEAN?" and give insights on how veOCEAN (vote-escrowed OCEAN) is used inside of Ocean Protocol to help advance the protocol.
+
+It may help for you to review the user guide on [how to estimate APY](../user-guides/data-farming/how-to-estimate-apy.md) to get a better idea for how long you'll want to lock your funds.  
+
+### Passive Rewards
+
+Passive Rewards is the first reward stream within Data Farming.
+
+At a minimum, Data Farmers earn [Passive Rewards](df-intro.md#what-are-passive-rewards) for locking their OCEAN.  
+
+It is by-design, without the need for anyone to put in any significant effort. You simply need to lock your tokens for a period of time.
 
-You will learn that by just holding veOCEAN passively, you are able to rewards.
+Passive Rewards offers a defined amount of rewards for each round that is divided amongst veOCEAN holders based on how much they hold.
 
-veOCEAN is a fork of veCRV. This enables participants to become a governance delegate, and eligible to receive rewards and engage with different protocol mechanisms.
+Once your tokens are locked up, you can then learn [how to claim rewards](../user-guides/data-farming/how-to-claim-rewards.md).
 
-## Data Farming
+### Active Rewards
 
-![DF Rewards Page](../.gitbook/assets/rewards/df_rewards_page.png)
+Active Rewards is the second reward streamm within Data Farming.
 
-[Data Farming 101](df-intro.md) introduces the different reward systems, how they work, and how to access them. By the end of the page, you should be more familiar with how Data Farming works and able to take next steps to curate assets.
+For participants that are more engaged and want to increase their yield, the [Active Rewards](df-intro.md#what-are-active-rewards) stream is broken down into multiple substreams. This offers contributors a variety of ways to help Ocean Protocol grow and increase your Data Farming rewards:
+1. Volume DF
+1. Challenge DF
 
```

Comment:
I would add a note here that the substreams for the df active rewards are subject to changes

Reply:
Nice catch. Added to here and emphasized it again inside of `df-intro.md`